src/etwlogger_driver.c

```diff
@@ -260,16 +289,28 @@ void etwlogger_log(LOG_CATEGORY log_category, const char* file, const char* func
     {
         switch (log_category)
         {
-            case AZ_LOG_INFO:
+            case AZ_LOG_ERROR:
```

Comment:
>AZ_LOG_ERROR [](start = 17, length = 12)

consider combining ERROR/WARNING and INFO/VERBOSE since this is just checking if there is a file/func/line param or not